driver: sensor: bma4xx: fix bus configuration init method name

demianzenkov opened the pull request:
'bma_spi_init' method was renamed & undeclared now. To fix project building using BMA4xx sensor (over SPI bus) method name should be changed to 'bma4xx_spi_init'

demianzenkov committed:
driver: sensor: bma4xx: clang-format

bma_4xx.c - clang-format

Signed-off-by: Demian Zenkov <zenkov94@mail.ru>

demianzenkov committed:
driver: sensor: bma4xx: fix bus init method name

'bma_spi_init' method is undeclared.
To fix building BMA4xx sensor driver (over SPI bus)
method name should be changed to 'bma4xx_spi_init'

Signed-off-by: Demian Zenkov <zenkov94@mail.ru>

yperess reviewed (changes requested):
Please run `clang-format` on your changed files

yperess reviewed (changes requested):
Sorry, the clang-format run made more changes that I expected. When that happens generally we ask that you split the PR into 2 commits (in the same PR).
* The first commit just runs clang-format
* The second commit adds your change

demianzenkov committed:
driver: sensor: bma4xx: fix bus init method name

'bma_spi_init' method is undeclared.
To fix building BMA4xx sensor driver (over SPI bus)
method name should be changed to 'bma4xx_spi_init'

Signed-off-by: Demian Zenkov <zenkov94@mail.ru>

demianzenkov committed:
Merge branch 'bma4xx_spi_config' of https://github.com/demianzenkov/zephyr into bma4xx_spi_config
